app/assets/stylesheets/app/quick_reservation.scss

```diff
@@ -0,0 +1,108 @@
+body.quick_reservation {
+  div#content {
+    h1 {
+      font-size: 2em;
+      border: none;
+    }
+    h1.gray {
+      color: #595959;
```

Comment:
This was `$navbarActiveBorderColor` but I changed it to get the accessibility spec to pass. Not sure if we want to update that variable value, or create an accessibility compliant alternative variable

Reply:
@giladshanan the change doesn't look very significant

Here's before

![before](https://github.com/tablexi/nucore-open/assets/624487/34e781c6-06bd-40d4-b80d-79c914a60b38)

And here's after

![after](https://github.com/tablexi/nucore-open/assets/624487/5b91921d-4485-4429-9e64-9ac283dac4df)